src/externalized/InventoryGraphicsModel.cc

```diff
@@ -0,0 +1,21 @@
+#include "InventoryGraphicsModel.h"
+
+InventoryGraphicsModel::InventoryGraphicsModel(SubImageModel small_, ST::string big_) : small(small_), big(big_) {
+}
+
+InventoryGraphicsModel InventoryGraphicsModel::deserialize(JsonObjectReader &obj) {
+    auto& smallSource = obj.GetValue("small");
+    ST::string big = obj.GetString("big");
+	JsonObjectReader smallReader(smallSource);
+    return InventoryGraphicsModel(SubImageModel::deserialize(smallReader), big);
+}
+
+JsonObject InventoryGraphicsModel::serialize(rapidjson::Document::AllocatorType& allocator) const {
+    JsonObject v(allocator);
+
+    auto s = small.serialize(allocator);
+    v.AddMember("small", s.getValue());
+    v.AddMember("big", big);
+
+    return v;
+}
```

Comment:
Didn't we standardize on tabs? Either way line 9 is misaligned.

Reply:
Yep we did. But my editor didn't know. Reindented the new files with tabs.

Reply:
Maybe time for an EditorConfig!